tests/e2e/tests/test_loadbalancer.py

```diff
@@ -0,0 +1,129 @@
+#
+# Copyright 2024 Canonical, Ltd.
+#
+import ipaddress
+import logging
+from pathlib import Path
+from typing import List
+
+import pytest
+from e2e_util import harness, util
+from e2e_util.config import MANIFESTS_DIR
+
+LOG = logging.getLogger(__name__)
+
+
+@pytest.mark.node_count(2)
+def test_loadbalancer(instances: List[harness.Instance]):
+    instance = instances[0]
+    tester_instance = instances[1]
+
+    lb_cidr = ""
+
+    p = instance.exec("ip -o -4 route show to default".split(" "), capture_output=True)
+    instance_default_ip = p.stdout.decode().split(" ")[8]
+
+    p = tester_instance.exec(
+        "ip -o -4 route show to default".split(" "), capture_output=True
```

Comment:
this is an attempt to figure out a CIDR for the loadbalancer, right? could you move it to a helper function with a comment?

Also, please prefer `["cmd", "paths"]` instead of `"cmd paths".split(" ")`

That way, we can return to it in the future (perhaps even set via config like TEST_LOADBALANCER_CIDR).